src/org/usfirst/frc/team2335/robot/commands/AutoDriveAway.java

```diff
@@ -0,0 +1,51 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class AutoDriveAway extends Command 
+{
+	private double _distance; //Creates distance passing parameter which is the distance that the ultrasonic needs to be from the wall
```

Comment:
Nitpick: Pull this line to the left; making two lines of commentary is better than having to scroll horizontally.